fed/mosaic-environment/src/main/java/org/eclipse/mosaic/fed/environment/ambassador/EnvironmentAmbassador.java

```diff
@@ -104,11 +104,11 @@ protected void processTimeAdvanceGrant(long time) throws InternalFederateExcepti
         for (CEvent event : config.events) {
```

Comment:
I would propose calling the iterating Object `eventConfig`. So we don't have the weird `event.event` calls

Reply:
@kschrab did you see this?

Reply:
Yes, but with the new flattened config object there are no such calls anymore.